### PR TITLE
[FSPE-5182] Changed semantic colors to support brighter and accessible text and icons

### PR DESCRIPTION
**Pull request by mattnickles**
#### Changed
* Changed: Icon semantic color values to brighter shades.
* Changed: Icon colors are now defined using icon utility color classes (e.g. `-icon--primary`) instead of text utility color classes (e.g. `-text--primary`).
* Changed: All component semantic states have been updated to render with new brighter semantic colors.
#### Fixed
* Fixed: Icon bug which displayed the SVG for icon-check-alt in place of icon-check.
#### Added
* Added: Color utility now includes documentation for customizing the color of icons.
#### Changed
* Changed: Color foundation has been updated with new semantic colors and usage guidelines.
* Changed: Icon documentation now includes usage guidelines for brand and semantic colors.
* Changed: Spinner documentation now includes usage guidelines for brand and semantic colors.
* Changed: Text utility documentation now includes usage guidelines for brand and semantic colors.
* Changed: Color utility documentation now includes usage guidelines for brand, brand complimentary, neutral, and semantic colors.
* Changed: All icons throughout docs have been changed from using text utility color classes to icon utility color classes.
#### Fixed
* Fixed: Template for authentication included an unnecessary size class that was causing alert content to misalign.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c2f41d957ee109cfd1b4fd3b92ac0562719ae192-chi-pr.enterprise.ctl.io" target="_blank"> https://c2f41d957ee109cfd1b4fd3b92ac0562719ae192-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1159/ for more details.

File: src/website/views/components/spinner/_examples.pug [hunk 1]
```diff
@@ -306,49 +308,81 @@ h2 Spinner Colors
...
+  | red (danger) for negative, yellow (warning) for needs attention, and grey (muted)
+  | for neutral. Semantic colors should never be used for decorative purposes.
+.example.-mb--3
+  .chi-grid.-no-gutter
+    .chi-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap
+      each color in ['success', 'warning', 'danger', 'muted']
+        div.-p--2.-text--center
+          chi-spinner(color=color, class='-m--3', size='md')
+  .example-tabs.-pl--2
+    ul.chi-tabs#examples-colors
```
Comment (jllr) [inline src/website/views/components/spinner/_examples.pug:355]: @mattnickles html tab in semantic colors is not loading properly

File: test/chi/components/card.pug [hunk 2]
```diff
@@ -159,7 +159,7 @@ title: Card
   h3 Empty
   .chi-card.-empty(style="width:20rem;")
     .chi-card__content
-      i.chi-icon.-md.-text--muted(class ='icon-circle-plus-outline')
+      i.chi-icon.-md.-icon--muted(class ='icon-circle-plus-outline')
```
Comment (jllr) [inline test/chi/components/card.pug:162]: @mattnickles we can remove the icon and change the caption to: This is an empty card, to be consistent with doc page

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1161/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1160/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://597d310255f3372a71205a67ba3001f1e2ff7b86-chi-pr.enterprise.ctl.io" target="_blank"> https://597d310255f3372a71205a67ba3001f1e2ff7b86-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1163/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://605718e0de25a90c38c30b7ad28941e5a12eeb22-chi-pr.enterprise.ctl.io" target="_blank"> https://605718e0de25a90c38c30b7ad28941e5a12eeb22-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1164/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://1e1b5160a824492a1c533d3394bc65dfcdbbd942-chi-pr.enterprise.ctl.io" target="_blank"> https://1e1b5160a824492a1c533d3394bc65dfcdbbd942-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1165/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-wwcrv6, https://ops.elasticbox.com/services/instances/i-ku6b8f, https://ops.elasticbox.com/services/instances/i-1t77v5, https://ops.elasticbox.com/services/instances/i-731x8p, https://ops.elasticbox.com/services/instances/i-b1ub2i

**Pull request merged**